staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile_test.go

```diff
@@ -70,31 +76,31 @@ password7,user7,uid7,"group1,group2",otherdata
 		{
 			Username: "user3",
 			Password: "password3",
-			User:     &user.DefaultInfo{Name: "user3", UID: "uid3", Groups: []string{"group1", "group2"}},
+			User:     &user.DefaultInfo{Name: "user3", UID: "uid3", Groups: []string{"group1", "group2"}, Tenant: "system"},
 			Ok:       true,
 		},
 		{
 			Username: "user4",
 			Password: "password4",
-			User:     &user.DefaultInfo{Name: "user4", UID: "uid4", Groups: []string{"group2"}},
+			User:     &user.DefaultInfo{Name: "user4", UID: "uid4", Groups: []string{"group2"}, Tenant: "system"},
 			Ok:       true,
 		},
 		{
 			Username: "user5",
 			Password: "password5",
-			User:     &user.DefaultInfo{Name: "user5", UID: "uid5", Groups: []string{"group5"}},
+			User:     &user.DefaultInfo{Name: "user5", UID: "uid5", Groups: []string{"group5"}, Tenant: "system"},
 			Ok:       true,
 		},
 		{
 			Username: "user6",
 			Password: "password6",
-			User:     &user.DefaultInfo{Name: "user6", UID: "uid6", Groups: []string{"group5"}},
+			User:     &user.DefaultInfo{Name: "user6", UID: "uid6", Groups: []string{"group5"}, Tenant: "system"},
 			Ok:       true,
 		},
 		{
 			Username: "user7",
 			Password: "password7",
-			User:     &user.DefaultInfo{Name: "user7", UID: "uid7", Groups: []string{"group1", "group2"}},
+			User:     &user.DefaultInfo{Name: "user7", UID: "uid7", Groups: []string{"group1", "group2"}, Tenant: "system"},
```

Comment:
Let's change the "system" to metav1.TenantSystem to align with passwordfile.go

Reply:
fixed.